### PR TITLE
Set up final BWC script URLs.

### PR DESCRIPTION
**Pull request by dzimine**
it says it all.

### CODE REVIEW

**Comment by emedvedev** [comment]
+1

**Comment by DoriftoShoes** [comment]
+1

File: docs/source/solutions/ipfabric/install.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ key you received when registering for evaluation or purchasing.
 
 .. code-block:: bash
 
-  wget https://brocade.com/bwc/install.sh && chmod +x install.sh
+  wget https://brocade.com/bwc/install/install.sh && chmod +x install.sh
```
Comment (LindsayHill) [inline docs/source/solutions/ipfabric/install.rst]: Seems a bit odd to use curl in one place, and wget in another

File: docs/source/solutions/ipfabric/install.rst [hunk 2]
```diff
@@ -17,22 +17,11 @@ run the following command, replacing
 
 .. code-block:: bash
 
-  curl -SsL https://brocade.com/bwc/install-suite.sh |  bash -s -- --user=st2admin --password=Ch@ngeMe --staging --unstable --suite=bwc-ipfabric-suite --license=${BWC_LISENSE_KEY}
+  curl -SsL https://brocade.com/bwc/install/install-suite.sh |  bash -s -- --user=st2admin --password=Ch@ngeMe --suite=bwc-ipfabric-suite --license=${BWC_LISENSE_KEY}
```
Comment (LindsayHill) [inline docs/source/solutions/ipfabric/install.rst]: s/LISENSE/LICENSE/

**Pull request merged**